Userland/Libraries/LibCore/DateTime.h

```diff
@@ -30,7 +30,7 @@ class DateTime {
     bool is_leap_year() const;
 
     void set_time(int year, int month = 1, int day = 1, int hour = 0, int minute = 0, int second = 0);
-    String to_string(const String& format = "%Y-%m-%d %H:%M:%S %Z") const;
+    String to_string(const String& format = "%Y-%m-%d %H:%M:%S") const;
```

Comment:
This is not related to your change, but I think the format string can just be a `StringView`.

Reply:
Yeah I've been seeing that a fair amount in older code recently, probably worth going through this stuff and span-ifying some things. Looks like there's only 2 methods with that issue in DateTime, so I can bang that out